colour/colorimetry/blackbody.py

```diff
@@ -594,3 +600,513 @@ def sd_blackbody(
         wavelengths,
         name=f"{temperature}K Blackbody",
     )
+
+
+def rayleigh_jeans_law(
+    wavelength,
+    temperature,
+) -> FloatingOrNDArray:
+    """
+    Returns the spectral radiance of electromagnetic radiation as a
```

Comment:
We are now using imperative docstring style, thus `Returns the ...` should be `Return the ...`